airflow/configuration.py

```diff
@@ -235,9 +229,28 @@ def getfloat(self, section, key):
         return float(self.get(section, key))
 
     def read(self, filenames):
-        ConfigParser.read(self, filenames)
+        super(AirflowConfigParser, self).read(filenames)
         self._validate()
 
+    def has_option(self, section, option):
+        try:
+            self.get(section, option)
```

Comment:
This is going to run any commands. How much would would be to check for the `_cmd` style instead -- i.e. to not call `.get()` here.

If a lot then add a comment saying why we are doing a get check and catching the error